AdvTicTacToe/Game/Business/Model/GameState.cs

```diff
@@ -0,0 +1,19 @@
+namespace AdvTicTacToe.Game.Business.Model;
+
+public class GameState
+{
+    public Board Board { get; set; } = null!;
+    public char BlankTile { get; set; }
+    public List<Player> Players { get; set; } = new();
+    public int CurrentPlayerId { get; set; }
+    public Player CurrentPlayer => Players[CurrentPlayerId];
+    public Player? Winner
```

Comment:
Personally, I don't think this is very readable. Also, this is a business rule whereas all the others are fields. Ideally, models should just contain fields.

Reply:
Hey Mario, was this in regard to line 9 or 10? What do you think a better approach would be? Thanks!